openedx/core/djangoapps/profile_images/views.py

```diff
@@ -121,9 +147,87 @@ def post(self, request, username):
         # send client response.
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def delete(self, request, username):  # pylint: disable=unused-argument
+        """
+        DELETE /api/user/v1/accounts/{username}/image
+        """
+
+        try:
+            # update the user account to reflect that the images were removed.
+            set_has_profile_image(username, False)
+
+            # remove physical files from storage.
+            profile_image_names = get_profile_image_names(username)
+            remove_profile_images(profile_image_names)
+
+            log.info(
+                LOG_MESSAGE_DELETE,
+                {'image_names': profile_image_names.values(), 'user_id': request.user.id}
+            )
+        except UserNotFound:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+        # send client response.
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class ProfileImageUploadView(APIView):
+    """
+    **DEPRECATION WARNING**
+
+        This view is deprecated.  Use POST
+        /api/user/v1/accounts/{username}/image instead.
+
```

Comment:
Would it be helpful to keep the example request around to show which URL routes are being deprecated?  We could make it part of the warning section, but I think it will be useful for API users to have access to a little more information about what is going away.  (DRF might provide this in the autogenerated parts of the docs.  I'm not sure.) @aleffert  or @mikekatz might have an opinion on this, too.